tic_tac_toe/game.py

```diff
@@ -63,42 +108,146 @@ def _check_winning_combinations(board, player):
     Returns the player (winner) of any of the winning combinations is completed
     by given player, or None otherwise.
     """
-    pass
-
+    winning_combos = [
+        [(0,0),(0,1),(0,2)],
+        [(1,0),(1,1),(1,2)],
+        [(2,0),(2,1),(2,2)],
+        [(0,0),(1,0),(2,0)],
+        [(0,1),(1,1),(2,1)],
+        [(0,2),(1,2),(2,2)],
+        [(0,0),(1,1),(2,2)],
+        [(0,2),(1,1),(2,0)]
+        ]        
+        
+    for combo in winning_combos:
+        if _is_winning_combination(board, combo, player) == True:
+            return player
+    return None
+            
+    
+    """
+    combination = []
+    num = 1
+    for i in range(3):
+        for j in range(3):
+            #print num
+            if board[i][j] == player:
+                combination.append(num)
+            num += 1
+        
+    return combination
+    """
+
+   
```

Comment:
I like how _is_winning_combination and _check_winning_combinations support each other, here.  Seems like a relatively simple solution.